packs/skype.yaml

```diff
@@ -1,172 +1,358 @@
+- name: shopping
+  src: http://static.skaip.org/img/emoticons/v2/ffffff/shopping.gif
+- name: goldmedal
+  src: http://static.skaip.org/img/emoticons/v2/ffffff/goldmedal.gif
+- name: fistbump
+  src: http://static.skaip.org/img/emoticons/v2/ffffff/fistbump.gif
+- name: goodluck
+  src: http://static.skaip.org/img/emoticons/v2/ffffff/goodluck.gif
+- name: gottarun
+  src: http://static.skaip.org/img/emoticons/v2/ffffff/gottarun.gif
+- name: celebrate
+  src: http://static.skaip.org/img/emoticons/v2/ffffff/handsinair.gif
+- name: handshake
+  src: http://static.skaip.org/img/emoticons/v2/ffffff/handshake.gif
+- name: hanukkah
+  src: http://static.skaip.org/img/emoticons/v2/ffffff/hanukkah.gif
+- name: heart
+  src: http://static.skaip.org/img/emoticons/v2/ffffff/heart.gif
+- name: hearthands
+  src: http://static.skaip.org/img/emoticons/v2/ffffff/hearthands.gif
+- name: hifive
+  src: http://static.skaip.org/img/emoticons/v2/ffffff/highfive.gif
+- name: holdon
+  src: http://static.skaip.org/img/emoticons/v2/ffffff/holdon.gif
+- name: holi
+  src: http://static.skaip.org/img/emoticons/v2/ffffff/holi.gif
+- name: holidayspirit
+  src: http://static.skaip.org/img/emoticons/v2/ffffff/holidayspirit.gif
+- name: henderson14
+  src: http://static.skaip.org/img/emoticons/v2/ffffff/henderson14.gif
+- name: joy
+  src: http://static.skaip.org/img/emoticons/v2/ffffff/joy.gif
+- name: klopp
+  src: http://static.skaip.org/img/emoticons/v2/ffffff/klopp.gif
+- name: skip
+  src: http://static.skaip.org/img/emoticons/v2/ffffff/skip.gif
+- name: key
+  src: http://static.skaip.org/img/emoticons/v2/ffffff/key.gif
+- name: kiss
+  src: http://static.skaip.org/img/emoticons/v2/ffffff/kiss.gif
+- name: lfcclap
+  src: http://static.skaip.org/img/emoticons/v2/ffffff/lfcclap.gif
+- name: lfcworried
+  src: http://static.skaip.org/img/emoticons/v2/ffffff/lfcworried.gif
+- name: letsmeet
+  src: http://static.skaip.org/img/emoticons/v2/ffffff/letsmeet.gif
+- name: like
+  src: http://static.skaip.org/img/emoticons/v2/ffffff/like.gif
+- name: lips
+  src: http://static.skaip.org/img/emoticons/v2/ffffff/lips.gif
+- name: lovegift
+  src: http://static.skaip.org/img/emoticons/v2/ffffff/lovegift.gif
+- name: man
+  src: http://static.skaip.org/img/emoticons/v2/ffffff/man.gif
+- name: mistletoe
+  src: http://static.skaip.org/img/emoticons/v2/ffffff/mistletoe.gif
+- name: ape
+  src: http://static.skaip.org/img/emoticons/v2/ffffff/monkey.gif
+- name: mooning
+  src: http://static.skaip.org/img/emoticons/v2/ffffff/mooning.gif
+- name: film
+  src: http://static.skaip.org/img/emoticons/v2/ffffff/movie.gif
+- name: movinghome
+  src: http://static.skaip.org/img/emoticons/v2/ffffff/movinghome.gif
+- name: '8'
+  src: http://static.skaip.org/img/emoticons/v2/ffffff/music.gif
+- name: lipsealed
+  src: http://static.skaip.org/img/emoticons/v2/ffffff/lipssealed.gif
+- name: ek
+  src: http://static.skaip.org/img/emoticons/v2/ffffff/naturescall.gif
+- name: b|
```

Comment:
`|` is not allowed, custom emoji names can only contain lower case letters, numbers, dashes and underscores.